AmplifyPlugins/Storage/Sources/AWSS3StoragePlugin/Support/Internal/StorageBackgroundEventsRegistry.swift

```diff
@@ -7,30 +7,44 @@
 
 import Foundation
 
+extension Notification.Name {
+    static let StorageBackgroundEventsRegistryWaiting = Notification.Name("StorageBackgroundEventsRegistryWaiting")
+}
+
 /// Background events registry.
 ///
 /// Discussion:
 ///    Multiple URLSession instances could be running background events with their own unique identifier. Those can be run
 ///    independently of the Amplify Storage plugin and this function will indiciate if it will handle the given identifier.
-class StorageBackgroundEventsRegistry {
```

Comment:
what is the point in making this an actor? I might be wrong, but does making this actor help in making the static variables protected against multiple access?

Reply:
I see that the static variables are converted to instance variables, resolving this comment.